src/utils.rs

```diff
@@ -210,6 +266,44 @@ mod tests {
             load_wallet(&wallet_path).unwrap();
         })
     }
+
+    #[test]
+    // case: wallet path not exist
+    fn test_ensure_no_wallet_exists_no_wallet() {
+        with_tmp_dir(|tmp_dir| {
+            let wallet_path = tmp_dir.join("wallet.json");
+            remove_wallet(&wallet_path);
+            ensure_no_wallet_exists(&wallet_path, false, &INPUT_NOP[..]).unwrap();
+        });
+    }
+
+    #[test]
+    #[should_panic]
```

Comment:
We can assert on the contents of the error:

`#[should_panic(expected = "Failed to create a new wallet at {} because a wallet already exists at that location.`")]`